File: src/main/java/com/bombproj/model/ProjectCode.java
package com.bombproj.model;

import com.queryflow.annotation.Table;
import lombok.Data;

import java.util.Date;

@Data
@Table("project_code")
public class ProjectCode {

    private String id;
    private String code;
    private String message;
    private String projectId;
    private Date createTime;
    private Date updateTime;
    private Integer state;

}

File: src/main/scala/com/bombproj/model/ProjectCode.scala
package com.bombproj.model

import java.util.Date

import com.queryflow.annotation.Table

import scala.beans.BeanProperty

@Table("project_code")
case class ProjectCode() {

    @BeanProperty var id: String = _
    @BeanProperty var code: String = _
    @BeanProperty var message: String = _
    @BeanProperty var projectId: String = _
    @BeanProperty var createTime: Date = _
    @BeanProperty var updateTime: Date = _
    @BeanProperty var state: Int = _

}
